### PR TITLE
adding class to block

### PR DESCRIPTION
**Pull request by itkg-rboessel**

### CODE REVIEW

File: DisplayBundle/Resources/views/Block/Email/sendEmailMessage.html.twig [hunk 1]
```diff
@@ -1 +1,5 @@
+{% extends 'OpenOrchestraDisplayBundle:Block:layout.html.twig' %}
+
+{% block content %}
 <p>{{messageSendEmail}}</p>
+{% endblock %}
```
Comment (nicolasThal) [inline DisplayBundle/Resources/views/Block/Email/sendEmailMessage.html.twig]: eof

File: DisplayBundle/Resources/views/Block/Footer/show.html.twig [hunk 2]
```diff
@@ -17,3 +19,4 @@
         {% endfor %}
     </div>
 {% endif %}
+{% endblock %}
```
Comment (nicolasThal) [inline DisplayBundle/Resources/views/Block/Footer/show.html.twig]: eof

File: DisplayBundle/Resources/views/Block/Footer/show.html.twig [hunk 3]
```diff
@@ -1,4 +1,6 @@
+{% extends 'OpenOrchestraDisplayBundle:Block:layout.html.twig' %}
 
+{% block content %}
 {% if element is defined %}
```
Comment (nicolasThal) [inline DisplayBundle/Resources/views/Block/Footer/show.html.twig]: indentation

File: DisplayBundle/Resources/views/Block/Email/sendEmailMessage.html.twig [hunk 4]
```diff
@@ -1 +1,5 @@
+{% extends 'OpenOrchestraDisplayBundle:Block:layout.html.twig' %}
+
+{% block content %}
 <p>{{messageSendEmail}}</p>
```
Comment (nicolasThal) [inline DisplayBundle/Resources/views/Block/Email/sendEmailMessage.html.twig]: indentation

File: DisplayBundle/Resources/views/Block/Gmap/show.html.twig [hunk 5]
```diff
@@ -1,5 +1,9 @@
+{% extends 'OpenOrchestraDisplayBundle:Block:layout.html.twig' %}
+
+{% block content %}
 <iframe width="425" height="350" frameborder="0" scrolling="no" marginheight="0" marginwidth="0" src="http://maps.google.fr/maps/ms?ie=UTF8&amp;msa=1&amp;ll={{ latitude }},{{ longitude }}&amp;spn=0.006173,0.013239&amp;t=m&amp;z={{ zoom }}&amp;output=embed"></iframe>
```
Comment (nicolasThal) [inline DisplayBundle/Resources/views/Block/Gmap/show.html.twig]: identation

File: DisplayBundle/Resources/views/Block/Menu/node.html.twig [hunk 6]
```diff
@@ -10,3 +13,4 @@
         {% endif %}
     </li>
 </ul>
+{%  endblock %}
```
Comment (nicolasThal) [inline DisplayBundle/Resources/views/Block/Menu/node.html.twig]: eof

**Comment by itkg-rboessel** [comment]
@nicolasThal @itkg-ngilain @itkg-nanne

**Comment by nicolasThal** [comment]
:+1:

**Pull request merged**